Resources/Prototypes/Entities/Mobs/Customization/Markings/gauze.yml

```diff
@@ -16,7 +16,7 @@
   id: GauzeLefteyePad
   bodyPart: Eyes
   markingCategory: Overlay
-  speciesRestriction: [Dwarf, Human, Reptilian, Arachnid, Rodentia]
```

Comment:
Why are you removing reptilians here?

Reply:
That is my bad, thanks for the check. Must have slipped through when I was going through the list. I'll get that fixed.